bitnami/supabase/templates/_helpers.tpl

```diff
@@ -22,8 +22,8 @@ Return the proper Supabase Studio Public URL
 Return the proper Supabase API Public URL
 */}}
 {{- define "supabase.api.publicURL" -}}
-{{- if .Values.publicURL -}}
-{{- print .Values.publicURL -}}
+{{- if .Values.global.publicURL -}}
```

Comment:
Shouldn't be at least a binding `.Values.global.publicURL` value in the _values.yaml_ file?

Reply:
You're right. You can reject this request. I had a missconfig in my own local values.